pkg/controller/common/nodenetwork.go

```diff
@@ -24,11 +23,10 @@ func NewNodeNetworkFromNode(node *corev1.Node, networkType networkv1.NetworkType
 	cn *networkv1.ClusterNetwork) *networkv1.NodeNetwork {
 	nn := &networkv1.NodeNetwork{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      node.Name + "-" + string(networkType),
-			Namespace: Namespace,
+			Name: node.Name + "-" + string(networkType),
 			OwnerReferences: []metav1.OwnerReference{
 				{
-					APIVersion: "ctlcorev1",
+					APIVersion: "core",
```

Comment:
Should it be `v1`? that's what we get from the Kubernetes core APIs

Reply:
Yes，it should be v1. It has been always wrong before.